### PR TITLE
Remove Github parenthetical from Help Wanted contributing link

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

**Comment by mansona** [comment]
> It should say more than just "Contribute" - "contribute to this page" or "contribute to the ecosystem" or something similar?

How about "Contribute to Ember" @MelSumner? It needs to be slightly more generic than "Contribute to this page" because it links to the help wanted app 🤔

File: addon/constants/links.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default [{
     type: 'link'
   }, {
     href: 'https://help-wanted.emberjs.com/',
-    name: 'Contribute (Github)',
+    name: 'Contribute',
```
Comment (locks) [inline addon/constants/links.js:78]: What if we just literally say Help Wanted for now and when we have the contributing guides we can revise?

```suggestion
    name: 'Help Wanted',
```

Comment (mansona) [reply addon/constants/links.js:78]: oooo I like that idea 🎉  @MelSumner how does that sound to you?

**Comment by locks** [comment]
CI is failing on beta and canary branches with the following message:

>             Error: Assertion Failed: Custom modifier managers must define their capabilities using the capabilities() helper function

**Pull request merged**